python/lsst/ts/externalscripts/auxtel/latiss_acquire_and_take_sequence.py

```diff
@@ -240,116 +292,188 @@ def set_metadata(self, metadata):
             gratings.add(grating)
         metadata.filter = f"{filters},{gratings}"
 
-    async def latiss_acquire(self, silent=True, doPointingModel=False):
+    async def latiss_acquire(self, doPointingModel=False):
+
+        # instantiate the quick measurement class
+        qm = QuickFrameMeasurement()
```

Comment:
consider making this a class attribute:

```
self.qm = QuickFrameMeasurement()
```

in the `__init__` method.

Reply:
So i'll do this, in fact I had it that way originally, except I thought that since it was only required in one of the functions that instantiating it for everything (when it might not get used) was not the right thing to do.
Is the "best practice" to put this in the `__init__` even if it won't be used?